tests/nested/manual/cloud-init-never-used-not-vuln/task.yaml

```diff
@@ -96,13 +96,9 @@ execute: |
     tests.nested exec "cloud-init status --wait"
 
     echo "Waiting for snapd to react to cloud-init"
-    for i in $(seq 60); do
-        if tests.nested exec "journalctl --no-pager -u snapd" | MATCH "Cloud-init reported"; then
-            break
-        fi
-        sleep 1
-    done
-
+    # It is needed || true because in ubuntu-core-16 there is not any cloud-init message in the journal log
+    tests.nested "retry --wait 1 -n 60 sh -c 'sudo journalctl --no-pager -u snapd | MATCH \"cloud-init reported\"'" || true
```

Comment:
this is unexpected, can you elaborate on this? Also note that the original check had "Cloud-init reported"... while in this check you are checking for "cloud-init reported"